runtime/vm/JFRChunkWriter.hpp

```diff
@@ -185,6 +186,7 @@ class VM_JFRChunkWriter {
 	static constexpr int THREAD_CONTEXT_SWITCH_RATE_SIZE = sizeof(float) + (3 * sizeof(I_64));
 	static constexpr int THREAD_STATISTICS_EVENT_SIZE = (6 * sizeof(U_64)) + sizeof(U_32);
 	static constexpr int THREAD_DUMP_EVENT_SIZE_PER_THREAD = 1000;
+	static constexpr int SYSTEM_PROCESS_EVENT_SIZE = sizeof(I_64) + sizeof(UDATA);
```

Comment:
The (estimated) size of an event should include the common fields (size and type); this should also allow up to 32 characters for the process identifier.